site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -174,9 +174,10 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.allocator.annotations`                      | [Annotations][annotations] added to the Agones allocator pods                                   | `{}`                   |
 | `agones.allocator.resources`                        | Allocator pods [resource requests/limit][resources]                                             | `{}`                   |
 | `agones.allocator.nodeSelector`                     | Allocator [node labels][nodeSelector] for pod assignment                                        | `{}`                   |
+| `agones.allocator.service.name`                     | Service name for the allocator                                                                  | `agones-allocator`     |
```

Comment:
For the new field, please add it to the table below (line 192) so that it doesn't appear in the docs until the next release. Thanks!